system-int-test/test/cmr/system_int_test/ingest/deleted_granules_index_test.clj

```diff
@@ -53,3 +71,110 @@
       (index/wait-until-indexed)
       (is (not (check-index-for-deleted-granule (:concept-id granule))))
       (dev-sys-util/eval-in-dev-sys `(concept-service/set-days-to-keep-tombstone! 365)))))
+
+(deftest deleted-granules-test-search
+  (let [collection-prov1 (d/ingest-umm-spec-collection
+                          "PROV1"
+                          (data-umm-c/collection {:EntryTitle "entry-title 1"
+                                                  :ShortName "S1"}))
+        collection-prov1-2 (d/ingest-umm-spec-collection
+                            "PROV1"
+                            (data-umm-c/collection {:EntryTitle "entry-title 2"
+                                                    :ShortName "S2"}))
+        collection-prov2 (d/ingest-umm-spec-collection "PROV2" (data-umm-c/collection {}))
+        granule1-prov1 (d/item->concept
+                        (dg/granule-with-umm-spec-collection
+                         collection-prov1 (:concept-id collection-prov1) {:concept-id "G1-PROV1"}))
+        granule2-prov1 (d/item->concept
+                        (dg/granule-with-umm-spec-collection
+                         collection-prov1 (:concept-id collection-prov1) {:concept-id "G2-PROV1"}))
+        granule3-prov1-2 (d/item->concept
+                          (dg/granule-with-umm-spec-collection
+                           collection-prov1-2 (:concept-id collection-prov1-2) {:concept-id "G3-PROV1"}))
+        granule4-prov2 (d/item->concept
+                        (assoc (dg/granule-with-umm-spec-collection
+                                 collection-prov2 (:concept-id collection-prov2) {:concept-id "G4-PROV2"})
+                               :provider-id "PROV2"))]
+    (ingest/ingest-concept granule1-prov1)
+    (ingest/delete-concept granule1-prov1)
+    (ingest/ingest-concept granule2-prov1)
+    (ingest/delete-concept granule2-prov1)
+    (ingest/ingest-concept granule3-prov1-2)
+    (ingest/delete-concept granule3-prov1-2)
+    (ingest/ingest-concept granule4-prov2)
+    (ingest/delete-concept granule4-prov2)
+    (index/wait-until-indexed)
+
+    (testing "Search for all"
+      (are3 [params granules]
+        (is (= (set (map :concept-id granules))
+               (set (find-deleted-granules params))))
+
+        "after tomorrow"
+        {:revision-date (t/plus- (t/now) (t/days 1))}
```

Comment:
I'm not sure what others think about this, but I always want to make sure we verify our query parameters on the API support underscores. I think we should be using `{"revision_date" (t/plus- (t/now) (t/days 1))}` or maybe just add one additional test for every query parameter that makes sure underscores are supported.